recipe/meta.yaml

```diff
@@ -45,6 +45,11 @@ requirements:
     - numbagg
     - flox
     - opt_einsum
+    - qt-niu
+    - pyvideoreader
```

Comment:
FY in pyproject.toml we restrict `pyvideoreader` to >0.5.3 (older versions won't work).